frontend/src/pages/Projects/Project/ProjectDetails/ProjectRegistrationSection/index.tsx

```diff
@@ -38,46 +42,132 @@ export const ProjectRegistrationSection = ({
+                });
+              }}
+              onCancel={async () => {
+                await lockRemover(query?.data?.dbRowLock).then(async () => {
+                  await query.refetch().then(() => {
+                    handleEditMode(false);
+                  });
+                });
+              }}
+              editFields={editFields()}
+            />
+          );
+          break;
+
+        case false: // not current user
+          content = (
+            <>
+              <Grid container spacing={2}>
+                <Grid item xs={6} md={6}>
+                  <Skeleton variant="rectangular" />
+                </Grid>
+                <Grid item xs={6} md={6}>
+                  <Skeleton variant="rectangular" />
+                </Grid>
+                <Grid item xs={6} md={6}>
+                  <LockPersonIcon />
+                  <h1>section locked for editing by: {query?.data?.dbRowLock.locked_by}</h1>
+                </Grid>
+                <Grid item xs={6} md={6}></Grid>
+                <Grid item xs={4} md={4}></Grid>
+                <Grid item xs={4} md={4}></Grid>
+                <Grid item xs={4} md={4}>
+                  {userHasEditCapability && (
+                    <>
+                      <FormEditButton
+                        buttonText="Take Over Editing"
+                        onClick={async () => {
+                          await lockRemover(query?.data?.dbRowLock).then(async () => {
+                            await handleDbLock(query, projectId).then(async () => {
+                              await query.refetch().then(() => {
+                                handleEditMode(true);
+                              });
+                            });
+                          });
+                        }}
+                      />
+                    </>
+                  )}
+                </Grid>
+              </Grid>
+            </>
+          );
+          break;
+      }
       break;
-    case true:
-      content = (
-        <EditForm
-          initialValues={query?.data?.data}
-          onSubmit={async (values) => {
-            return handleUpdate({
-              changedValues: values,
-              currentRowData: query?.data?.data,
-              apiUrl: `projects/${projectId}`,
-              handleEditMode: setEditMode,
-              queryKeys: [`project - ${projectId}`],
-              successMessage: `Changes saved successfully for project ${projectId}`,
-              errorMessage: `There was an issue saving your changes for project ${projectId}`,
-            });
-          }}
-          onCancel={() => {
-            setEditMode(false);
-          }}
-          editFields={editFields()}
-        />
-      );
+
+    case false: //db row is not locked  - query?.data?.dbRowLock?.locked
+      switch (editMode) {
+        case false: //  db row is not locked & not edit mode
+          content = (
+            <>
```

Comment:
I don't think we need <>s if there's only one element inside.

Reply:
yeah it should work without the wrapper for sure, since formEditButton is probably a div anyway